### PR TITLE
Merge polkadot-0.8.25

### PR DESCRIPTION
**Pull request by AndreiNavoichyk**

### CODE REVIEW

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -116,7 +116,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 279,
-	impl_version: 0,
+	impl_version: 1,
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/src/lib.rs:119]: Why do we increase `impl_version` only?

File: primitives/npos-elections/src/mock.rs [hunk 2]
```diff
@@ -314,7 +309,7 @@ pub fn check_assignments_sum<T: PerThing>(assignments: Vec<Assignment<AccountId,
 	for Assignment { distribution, .. } in assignments {
 		let mut sum: u128 = Zero::zero();
 		distribution.iter().for_each(|(_, p)| sum += p.deconstruct().saturated_into::<u128>());
-		assert_eq_error_rate!(sum, T::ACCURACY.saturated_into::<u128>(), 1);
+		assert_eq!(sum, T::ACCURACY.saturated_into::<u128>(), "Assignment ratio sum is not 100%");
```
Comment (AndreiNavoichyk) [inline primitives/npos-elections/src/mock.rs:312]: Why should we use `assert_eq`?

**Review by AndreiNavoichyk — COMMENTED** [review]
Share in the description the link to the ddc-pallet repo's branch. We need to review it as well.

**Review by AndreiNavoichyk — COMMENTED** [review]
@Raid5594 
Please, merge the latest dev-cere

**Review by Raid5594 — APPROVED** [review]

File: bin/node/runtime/src/lib.rs (hunk 1 above)
Comment (Raid5594) [reply bin/node/runtime/src/lib.rs:119]: Following the increase of implementation version in the target commit used for merge

File: primitives/npos-elections/src/mock.rs (hunk 2 above)
Comment (Raid5594) [reply primitives/npos-elections/src/mock.rs:312]: Tests are failing otherwise. That syntax is actually used in the target commit

**Pull request merged**